=== Suite started at 2016-01-25 20:15:56
=== Starting test, 11 test cases
=cases         11
=user          travis
=host          testing-gce-6f93673d-a5c0-4a2f-bc4d-289fe831f05b
=hosts         testing-gce-6f93673d-a5c0-4a2f-bc4d-289fe831f05b 
=emulator_vsn  6.4
=emulator      beam
=otp_release   17
=started       2016-01-25 20:15:56
=case          oauth_SUITE:init_per_suite
=logfile       oauth_suite.init_per_suite.html
=started       2016-01-25 20:15:56
=ended         2016-01-25 20:15:57
=result        ok
=elapsed       0.006258
=case          oauth_SUITE:init_per_group
=logfile       oauth_suite.init_per_group.html
=group_props   [{name,token_login},sequence]
=started       2016-01-25 20:15:57
=ended         2016-01-25 20:15:57
=result        ok
=elapsed       0.510726
=case          oauth_SUITE:request_tokens_test
=logfile       oauth_suite.request_tokens_test.html
=started       2016-01-25 20:15:57
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:15:57
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 1 of 11 ***
=case          oauth_SUITE:login_access_token_test
=logfile       oauth_suite.login_access_token_test.html
=started       2016-01-25 20:15:57
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:15:57
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 2 of 11 ***
=case          oauth_SUITE:login_refresh_token_test
=logfile       oauth_suite.login_refresh_token_test.html
=started       2016-01-25 20:15:57
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:15:57
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 3 of 11 ***
=case          oauth_SUITE:login_with_other_users_token
=logfile       oauth_suite.login_with_other_users_token.html
=started       2016-01-25 20:15:57
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:15:57
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 4 of 11 ***
=case          oauth_SUITE:login_with_malformed_token
=logfile       oauth_suite.login_with_malformed_token.html
=started       2016-01-25 20:15:57
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:15:57
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 5 of 11 ***
=case          oauth_SUITE:end_per_group
=logfile       oauth_suite.end_per_group.html
=group_props   [{name,token_login},sequence]
=started       2016-01-25 20:15:57
=ended         2016-01-25 20:15:58
=result        ok
=elapsed       0.921243
=group_time    0.671s
=case          oauth_SUITE:init_per_group
=logfile       oauth_suite.init_per_group.752958.635.html
=group_props   [{name,token_revocation},sequence]
=started       2016-01-25 20:15:58
=ended         2016-01-25 20:15:59
=result        ok
=elapsed       0.509478
=case          oauth_SUITE:login_with_revoked_token_test
=logfile       oauth_suite.login_with_revoked_token_test.html
=started       2016-01-25 20:15:59
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:15:59
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 6 of 11 ***
=case          oauth_SUITE:token_revocation_test
=logfile       oauth_suite.token_revocation_test.html
=started       2016-01-25 20:15:59
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:15:59
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 7 of 11 ***
=case          oauth_SUITE:end_per_group
=logfile       oauth_suite.end_per_group.752959.226.html
=group_props   [{name,token_revocation},sequence]
=started       2016-01-25 20:15:59
=ended         2016-01-25 20:16:00
=result        ok
=elapsed       0.922142
=group_time    0.592s
=case          oauth_SUITE:init_per_group
=logfile       oauth_suite.init_per_group.752960.174.html
=group_props   [{name,provision_token}]
=started       2016-01-25 20:16:00
=ended         2016-01-25 20:16:00
=result        ok
=elapsed       0.511325
=case          oauth_SUITE:provision_token_login
=logfile       oauth_suite.provision_token_login.html
=started       2016-01-25 20:16:00
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:16:00
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 8 of 11 ***
=case          oauth_SUITE:end_per_group
=logfile       oauth_suite.end_per_group.752960.744.html
=group_props   [{name,provision_token}]
=started       2016-01-25 20:16:00
=ended         2016-01-25 20:16:01
=result        ok
=elapsed       0.924381
=group_time    0.570s
=case          oauth_SUITE:init_per_group
=logfile       oauth_suite.init_per_group.752961.694.html
=group_props   [{name,commands}]
=started       2016-01-25 20:16:01
=ended         2016-01-25 20:16:02
=result        ok
=elapsed       0.513292
=case          oauth_SUITE:revoke_token_cmd_when_no_token
=logfile       oauth_suite.revoke_token_cmd_when_no_token.html
=started       2016-01-25 20:16:02
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:16:02
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 9 of 11 ***
=case          oauth_SUITE:revoke_token_cmd
=logfile       oauth_suite.revoke_token_cmd.html
=started       2016-01-25 20:16:02
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:16:02
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 10 of 11 ***
=case          oauth_SUITE:end_per_group
=logfile       oauth_suite.end_per_group.752962.297.html
=group_props   [{name,commands}]
=started       2016-01-25 20:16:02
=ended         2016-01-25 20:16:03
=result        ok
=elapsed       0.924559
=group_time    0.603s
=case          oauth_SUITE:init_per_group
=logfile       oauth_suite.init_per_group.752963.255.html
=group_props   [{name,cleanup}]
=started       2016-01-25 20:16:03
=ended         2016-01-25 20:16:03
=result        ok
=elapsed       0.512366
=case          oauth_SUITE:token_removed_on_user_removal
=logfile       oauth_suite.token_removed_on_user_removal.html
=started       2016-01-25 20:16:03
=== ERROR! init_per_testcase crashed!
	Location: [{oauth_SUITE,clean_token_db,<a href="oauth_suite.src.html#362">362</a>},
              {oauth_SUITE,init_per_testcase,<a href="oauth_suite.src.html#99">99</a>},
              {test_server,do_init_per_testcase,1233},
              {test_server,run_test_case_eval1,997},
              {test_server,run_test_case_eval,968}]
	Reason: {{badmatch,{error,"#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                 [{oauth_SUITE,clean_token_db,0,
                               [{file,"oauth_SUITE.erl"},{line,362}]},
                  {oauth_SUITE,init_per_testcase,2,
                               [{file,"oauth_SUITE.erl"},{line,99}]},
                  {test_server,do_init_per_testcase,2,
                               [{file,"test_server.erl"},{line,1233}]},
                  {test_server,run_test_case_eval1,6,
                               [{file,"test_server.erl"},{line,997}]},
                  {test_server,run_test_case_eval,9,
                               [{file,"test_server.erl"},{line,968}]}]}

=ended         2016-01-25 20:16:03
=result        auto_skipped: {failed,
                              {oauth_SUITE,init_per_testcase,
                               {{badmatch,
                                 {error,
                                  "#42S02Table 'ejabberd.auth_token' doesn't exist"}},
                                [{oauth_SUITE,clean_token_db,0,
                                  [{file,"oauth_SUITE.erl"},{line,362}]},
                                 {oauth_SUITE,init_per_testcase,2,
                                  [{file,"oauth_SUITE.erl"},{line,99}]},
                                 {test_server,do_init_per_testcase,2,
                                  [{file,"test_server.erl"},{line,1233}]},
                                 {test_server,run_test_case_eval1,6,
                                  [{file,"test_server.erl"},{line,997}]},
                                 {test_server,run_test_case_eval,9,
                                  [{file,"test_server.erl"},{line,968}]}]}}}
=== *** SKIPPED test case 11 of 11 ***
=case          oauth_SUITE:end_per_group
=logfile       oauth_suite.end_per_group.752963.833.html
=group_props   [{name,cleanup}]
=started       2016-01-25 20:16:03
=ended         2016-01-25 20:16:04
=result        ok
=elapsed       0.463304
=group_time    0.577s
=case          oauth_SUITE:end_per_suite
=logfile       oauth_suite.end_per_suite.html
=started       2016-01-25 20:16:04
=ended         2016-01-25 20:16:04
=result        ok
=elapsed       0.036714
=group_time    7.379s
=== TEST COMPLETE, 0 ok, 0 failed, 11 skipped of 11 test cases

=finished      2016-01-25 20:16:04
=failed        0
=successful    0
=user_skipped  0
=auto_skipped  11
